app/controllers/software_records_controller.rb

```diff
@@ -182,6 +182,14 @@ def software_record_params
       :cm_stakeholders,
       :cm_other_notes,
       :sensitive_information,
+      :qa_url,
+      :dev_url,
+      :production_server,
```

Comment:
Is this field for prod server url? Probably should match pattern with or url fields above

Reply:
Yeah and I think they need to be repeatable to represent a full stack application.